### DISCUSSION TITLE
How to pass extra data in params?

### DISCUSSION THREAD

**Discussion by yogeshgalav** [vuejs/router#2198] [Help and Questions] [answered]
In vue2 we were able to pass lot of information including object hidden in router params,
I know params are made for url parameters but still the old application or old developers has used the loop hole in vue2 router
and passed the lot of information in every other page with vue router instead of vuex
It is now giving problem in vue3,Is there anyhow it could support that old method?

**Comment by skirtles-code** [comment, marked as answer]
There are some suggestions for ways to handle this at <https://github.com/vuejs/router/blob/main/packages/router/CHANGELOG.md#414-2022-08-22>.
